diff --git a/cocotb/cocotb.bzl b/cocotb/cocotb.bzl
--- a/cocotb/cocotb.bzl
+++ b/cocotb/cocotb.bzl
@@ -134,2 +134,2 @@
     test_module_args = _pymodules_to_argstring(ctx.files.test_module, "test_module")
-    python_interpreter = ctx.toolchains["@rules_python//python:toolchain_type"].py3_runtime.interpreter.path
+    python_interpreter = ctx.toolchains["@rules_python//python:toolchain_type"].py3_runtime.interpreter.short_path
